.github/actions/javascript/getGraphiteString/getGraphiteString.ts

```diff
@@ -1,49 +1,49 @@
 import * as core from '@actions/core';
-import * as github from '@actions/github';
 import fs from 'fs';
 
 const run = () => {
     // Prefix path to the graphite metric
     const GRAPHITE_PATH = 'reassure';
+    const PR_NUMBER = core.getInput('PR_NUMBER', {required: true});
 
-    let regressionOutput;
-    try {
-        regressionOutput = JSON.parse(fs.readFileSync('.reassure/output.json', 'utf8'));
-    } catch (err) {
-        // Handle errors that occur during file reading or parsing
-        if (err instanceof Error) {
-            console.error('Error while parsing output.json:', err.message);
-            core.setFailed(err);
-        }
-    }
+    // Read the contents of the file, the file is in the JSONL format
+    const regressionFile = fs.readFileSync('.reassure/baseline.perf', 'utf8');
+
+    // Split file contents by newline to get individual JSON entries
+    const regressionEntries = regressionFile.split('\n');
 
-    const creationDate = regressionOutput.metadata.current.creationDate;
-    const timestampInMili = new Date(creationDate).getTime();
+    // Initialize string to store Graphite metrics
+    let graphiteString = '';
 
-    // Graphite accepts timestamp in seconds
-    const timestamp = Math.floor(timestampInMili / 1000);
+    // Iterate over each entry
+    regressionEntries.forEach((entry) => {
+        // Skip empty lines
+        if (entry.trim() === '') {
+            return;
+        }
 
-    // Get PR number from the github context
-    const prNumber = github.context.payload.pull_request?.number;
+        try {
+            const current = JSON.parse(entry);
 
-    // We need to combine all tests from the 4 buckets
-    const reassureTests = [...regressionOutput.meaningless, ...regressionOutput.significant, ...regressionOutput.countChanged, ...regressionOutput.added];
+            // Extract timestamp, Graphite accepts timestamp in seconds
+            const timestamp = current.metadata?.creationDate ? Math.floor(new Date(current.metadata.creationDate).getTime() / 1000) : '';
 
-    // Map through every test and create string for meanDuration and meanCount
-    // eslint-disable-next-line rulesdir/prefer-underscore-method
-    const graphiteString = reassureTests
-        .map((test) => {
-            const current = test.current;
-            // Graphite doesn't accept metrics name with space, we replace spaces with "-"
-            const formattedName = current.name.split(' ').join('-');
+            if (current.name && current.meanDuration && current.meanCount) {
```

Comment:
Should we also add the timestamp to this condition? I assume we should not create the graphite string if the timestamp is missing for whatever reason

Reply:
good point, I just added the check